### DISCUSSION TITLE
How to generate special Chinese questions? like `剑指 Offer II 001` and `面试题 01.01`

### DISCUSSION THREAD

**Discussion by caiddd** [j178/leetgo#69] [Q&A] [answered]
They're mostly at the back of the question bank, like [this](https://leetcode.cn/problems/Db3wC1/l) and [this](https://leetcode.cn/problems/cong-shang-dao-xia-da-yin-er-cha-shu-lcof/).

**Comment by j178** [comment, marked as answer]
Hi, thanks for opening this issue.

There is indeed a issue in generating questions like 'LCP 31' and '剑指 Offer 32 - I', it should be fixed in v0.1.5, please upgrade and try again.

By the way, it should be easier to pick these special questions by using their slug (the last part of the problem URL):
![image](https://user-images.githubusercontent.com/10510431/215301880-6f860b8b-eea5-4761-8b87-8b13393e76a3.png)
![image](https://user-images.githubusercontent.com/10510431/215301949-6713fc58-c08f-4f41-b837-71280627999e.png)
